src/compiler/compile/utils/a11y.ts

```diff
@@ -223,3 +224,111 @@ export function is_semantic_role_element(role: ARIARoleDefinitionKey, tag_name:
 	}
 	return false;
 }
+
+// https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofilling-form-controls:-the-autocomplete-attribute
+const address_type_tokens = new Set(['shipping', 'billing']);
+const autofill_field_name_tokens = new Set([
+	'name',
+	'honorific-prefix',
+	'given-name',
+	'additional-name',
+	'family-name',
+	'honorific-suffix',
+	'nickname',
+	'username',
+	'new-password',
+	'current-password',
+	'one-time-code',
+	'organization-title',
+	'organization',
+	'street-address',
+	'address-line1',
+	'address-line2',
+	'address-line3',
+	'address-level4',
+	'address-level3',
+	'address-level2',
+	'address-level1',
+	'country',
+	'country-name',
+	'postal-code',
+	'cc-name',
+	'cc-given-name',
+	'cc-additional-name',
+	'cc-family-name',
+	'cc-number',
+	'cc-exp',
+	'cc-exp-month',
+	'cc-exp-year',
+	'cc-csc',
+	'cc-type',
+	'transaction-currency',
+	'transaction-amount',
+	'language',
+	'bday',
+	'bday-day',
+	'bday-month',
+	'bday-year',
+	'sex',
+	'url',
+	'photo'
+]);
+const contact_type_tokens = new Set(['home', 'work', 'mobile', 'fax', 'pager']);
+const autofill_contact_field_name_tokens = new Set([
+	'tel',
+	'tel-country-code',
+	'tel-national',
+	'tel-area-code',
+	'tel-local',
+	'tel-local-prefix',
+	'tel-local-suffix',
+	'tel-extension',
+	'email',
+	'impp'
+]);
+
+export function is_valid_autocomplete(type: null | true | string, autocomplete: null | true | string) {
+	if (typeof autocomplete !== 'string' || typeof type !== 'string') {
+		return false;
+	}
+
+	const tokens = autocomplete.trim().toLowerCase().split(regex_whitespaces);
+	const normalized_type = type.toLowerCase();
+
+	const input_wears_autofill_anchor_mantle = normalized_type === 'hidden';
+	const input_wears_autofill_expectation_mantle = !input_wears_autofill_anchor_mantle;
```

Comment:
The language comes from [the spec][spec]. Given that [`axe-core`’s implementation][axe-implementation] does not implement this part of the spec, do you want me to remove it?

[spec]: https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofilling-form-controls:-the-autocomplete-attribute
[axe-implementation]: https://github.com/dequelabs/axe-core/blob/c7957d2e8be4bfd5c9aa678036fb945dd23a9c27/lib/commons/text/is-valid-autocomplete.js

Reply:
What exactly do you mean by that? If it's about simplifying this to always allow on and off, then yes, let's relax this